src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -746,12 +742,15 @@ def notifyLargeData(self, aboveWarningThreshold, transferId, wflowName, dataSize
         """
         # Warn about data transfer subscriptions going above some threshold
         if aboveWarningThreshold:
-            emailSubject = "[MS] Large pending data transfer under request id: {}".format(transferId)
-            emailMsg = "Workflow: {}\nhas a large amount of ".format(wflowName)
-            emailMsg += "data subscribed: {} TB,\n".format(teraBytes(dataSize))
-            emailMsg += "for {} data: {}.""".format(dataIn['type'], dataIn['name'])
-            self.emailAlert.send(emailSubject, emailMsg)
-            self.logger.info(emailMsg)
+            alertName = "ms-transferor: Transfer over threshold: {}".format(transferId)
+            alertSeverity = "high"
+            alertSummary = "[MS] Large pending data transfer under request id: {}".format(transferId)
+            alertDescription = "Workflow: {}\nhas a large amount of ".format(wflowName)
+            alertDescription += "data subscribed: {} TB,\n".format(teraBytes(dataSize))
+            alertDescription += "for {} data: {}.""".format(dataIn['type'], dataIn['name'])
+
+            self.alertManagerApi.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, "ms-transferor")
```

Comment:
Erik, and the same comment goes here (about the instance variable containing the service name).